### ISSUE TITLE
Can't get the realize package

### ISSUE THREAD

**Issue by yyh-gl** [oxequa/realize#262]
# Problem

When I build the docker image, I faced with a below problem.

```
< Omitted >

Step 9/11 : RUN go get github.com/oxequa/realize
 ---> Running in 615cd3c98668
go: finding github.com/oxequa/realize v2.0.2+incompatible
go: downloading github.com/oxequa/realize v2.0.2+incompatible
go: extracting github.com/oxequa/realize v2.0.2+incompatible
go: downloading golang.org/x/net v0.0.0-20190923162816-aa69164e4478
go: downloading golang.org/x/crypto v0.0.0-20190923035154-9ee001bba392
go: finding github.com/oxequa/interact latest
go: extracting golang.org/x/crypto v0.0.0-20190923035154-9ee001bba392
go: finding github.com/go-siris/siris/core/errors latest
go: extracting golang.org/x/net v0.0.0-20190923162816-aa69164e4478
go: finding gopkg.in/urfave/cli.v2 v2.0.0
go: downloading github.com/oxequa/interact v0.0.0-20171114182912-f8fb5795b5d7
go: extracting github.com/oxequa/interact v0.0.0-20171114182912-f8fb5795b5d7
go: downloading gopkg.in/urfave/cli.v2 v2.0.0
go: finding github.com/go-siris/siris/core latest
go: finding github.com/go-siris/siris v7.4.0+incompatible
go: downloading github.com/go-siris/siris v7.4.0+incompatible
go: extracting github.com/go-siris/siris v7.4.0+incompatible
go: finding github.com/satori/go.uuid v1.2.0
go: downloading github.com/satori/go.uuid v1.2.0
go: extracting github.com/satori/go.uuid v1.2.0
build github.com/oxequa/realize: cannot load gopkg.in/urfave/cli.v2: cannot find module providing package gopkg.in/urfave/cli.v2
ERROR: Service 'app' failed to build: The command '/bin/sh -c go get github.com/oxequa/realize' returned a non-zero code: 1
```

# Environment
- macOS Mojave 10.14.6
- Golang: 1.12.5

# Proposal solution

I propose to this problem below solution.
PR is [here](https://github.com/oxequa/realize/pull/261).

---

I changed import path of `urfave/cli` like below and fixed it.

Before: `"gopkg.in/urfave/cli.v2"`
After: `"github.com/urfave/cli/v2"`

**Comment by akalp** [comment]
I added it but I still can't run it. There is no executable file of "realize" in the bin directory.

> add this line in go.mod
> `replace gopkg.in/urfave/cli.v2 => github.com/urfave/cli/v2 v2.2.0`
> it's works

**Comment by yyh-gl** [comment]
@akalp 
Did you execute command below?
`$ go build -o ./<executable file name>`

Note: The executable file name must be something other than "realize".

**Comment by yyh-gl** [comment]
I submitted new issue https://github.com/oxequa/realize/issues/266.
Please check that.
